flight/Modules/System/systemmod.c

```diff
@@ -1095,6 +1107,9 @@ static uint32_t processPeriodicUpdates()
 	return timeToNextUpdate - now;
 }
 
+DONT_BUILD_IF(ANNUNCIATORSETTINGS_MANUALBUZZER_MAXOPTVAL >
+	MANUALCONTROLCOMMAND_ACCESSORY_NUMELEM, TooManyManualBuzzers);
```

Comment:
Arguably want to check not equal, to make sure we support all the acc chans in future?